### ISSUE TITLE
a question 

### ISSUE THREAD

**Issue by Rebilium** [huskcasaca/effortless#75]
Great mod. features ... Unique
however i can't find anything to explain in layman terms how to use it, what buttons to press. for noobs or old timers like me.
one of the features is magnet and randomizer and i can't find a tool or a button to use it or do its thing. can't find any info to know what to do. i was able to find the pattern settings which was by accessing the key binds.
im sorry if i sound rude or offensive but i really like your mod and i hate the fact im unable to operate it due to my ignorance. all i ask is simple words to operate the mod.
wish nothing but the best in life and best of luck!

**Comment by AndiLeni** [comment]
Yep, same. There are not docs or anything else which might help.
I can access the pattern/randomizer settings, but I have no idea how to actually use it. 
Were you able to figure this out @Rebilium ?

**Comment by Rebilium** [comment]
@AndiLeni i checked the key binds and then just had to figure things out by trail and error 😅😂. I also figured out some stuff just by using it. It's an amazing mod i just hope that it won't get abandoned.

**Comment by AndiLeni** [comment]
I think this issue can be closed now, as there is a nice description in the readme :-)
